.gitlab-ci.yml

```diff
@@ -9,14 +9,15 @@ variables:
     EXCLUDE_PYTHON: 311
     PYTHON_VERSION: "registry.git.rwth-aachen.de/ebc/ebc_all/gitlab_ci/templates:python_3.9"
     PAGES_BRANCH: master
+    PYTHON_PACKAGE_NAME: "filip"
 
 include:
     - project: 'EBC/EBC_all/gitlab_ci/templates'
       file: 'python/code-quality/pylint.gitlab-ci.yml'
     - project: 'EBC/EBC_all/gitlab_ci/templates'
       file: 'python/doc/sphinxdoc.gitlab-ci.yml'
     - project: 'EBC/EBC_all/gitlab_ci/templates'
-      file: 'pages/pages.gitlab-ci.yml'
+      file: 'pages/gl-pages.gitlab-ci.yml'
```

Comment:
This needs to be changed to `gh-pages` for  the CI to work on Github. With that the CI should not throw any errors.
With that fixed I think we can merge!